src/librustc_trait_selection/traits/error_reporting/suggestions.rs

```diff
@@ -1087,6 +1089,18 @@ impl<'a, 'tcx> InferCtxtExt<'tcx> for InferCtxt<'a, 'tcx> {
             }
         };
 
+        let generator_body = self
```

Comment:
```suggestion
        let hir = self.tcx.hir();
        let generator_body = self
```
(It's used quite a bit in this method, so replacing the uses with the binding would clean things up a bit.)